Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -93,146 +71,112 @@ MetsEditorMissingFile=Die Datei {0} kann im Ordner {1} nicht gefunden werden.
 abbrechen=Abbrechen
 abgeschlosseneTasksAusListeEntfernen=Abgeschlossene Aufgaben l\u00F6schen
-abschliessen=Abschlie\u00DFen
 addDemoTask=Beispielaufgabe hinzuf\u00FCgen
 addFilter=Filter hinzuf\u00FCgen
 addNewMetadataGroup=Neue Metadatengruppe hinzuf\u00FCgen
 addToBatch=Vorgang wurde zur Gruppe {0} hinzugef\u00FCgt.
 addToProcessLog=Zu Vorgangslog hinzuf\u00FCgen
 addToSelectedBatch=Selektierte Vorg\u00E4nge zu ausgew\u00E4hltem Batch hinzuf\u00FCgen
 administration=Administration
+# administrativ is supposedly use for translation of TaskEditType
 administrativ=administrativ
-aktivSeit=Aktiv seit
 aktiveBenutzer=Aktive Benutzer
 aktualisierungstyp=Art der Aktualisierung
 aktuelleProzesse=Vorg\u00E4nge
 aktuelleSchritte=Meine Aufgaben
-aktuelleZeit=Aktuelle Zeit
 aktuellerOrdner=Aktueller Ordner
-aktuellesPasswortFalsch=Das aktuelle Passwort ist falsch
 allActiveUsers=alle aktiven Nutzer
 alleAuswaehlen=Alle ausw\u00E4hlen
 alleSeiten=Alle Seiten
 alleTasksAusListeEntfernen=Alle Aufgaben l\u00F6schen
 alleTrefferDieserSeiteInMeinHomeverzeichnis=Alle Treffer dieser Seite in mein Homeverzeichnis laden
-allgemeineEigenschaften=Allgemeine Eigenschaften
 allgemeines=Allgemeines
-allgemeinesText=Die Software Kitodo dient zur Organisation von Digitalisierungsworkflows. Das Softwarepaket f\u00FCr diese kooperative und kollaborative Arbeitsumgebung ist der zentrale Baustein und erf\u00FCllt folgende Funktionalit\u00E4ten\: Plattform-Unabh\u00E4ngigkeit (Web-Applikation), da Partner weltweit auf das Tool zugreifen m\u00FCssen; Zentrale Metadaten-Verwaltung, d.h. Katalogisieren und Vervollst\u00E4ndigung der Metadaten von verschiedenen Standorten aus (zum Beispiel Erstellung der russischen Metadaten in Moskau, Transliterationen in Hannover); Zentrale Verwaltung der Digitalisate (Images); Import- und Exportschnittstellen f\u00FCr Metadaten und f\u00FCr Fremd-Digitalisate (zum Beispiel aus Russland); Controlling-Mechanismen\: welcher Partner ist wie weit mit seinen Arbeiten, welche Zeitschrift ist in welchem Bearbeitungsstadium etc.; Fehlermeldungen, Abschlie\u00DFen eines Arbeitsschrittes und Weiterleiten zur n\u00E4chsten Arbeitsstufe (eventuell hier auch Wechsel/\u00DCbergabe von einem Partner zum anderen Partner). Komponenten f\u00FCr das verteilte Workflowmanagement sind integriert worden, um die Verwaltung einer verteilten Produktion und Kommunikation zwischen den unterschiedlichen Partnern zu gew\u00E4hrleisten. F\u00FCr jeden einzelnen Workflowschritt kann eingestellt werden, ob er parallel zum vorherigen durchgef\u00FChrt werden kann oder erst nach dessen erfolgreichem Abschluss. Ferner werden Korrekturschleifen abgebildet, um in sp\u00E4teren Schritten festgestellte fehlerhafte Daten fr\u00FCherer Schritte zu vermerken. Die Applikation ist transparent konzipiert, somit ist es f\u00FCr jeden Projektpartner jederzeit m\u00F6glich, den Status der Teilschritte des Workflows inklusive Personenzuordnung f\u00FCr einzelne Werke detailliert einzusehen.
```

Comment:
This is used in `kitodo_config.properties` file as one of two standard possible values.